data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -64,6 +65,27 @@ def _climate_factor(self) -> bool:
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
+
+        # I am thinking that we make constants for these since these thresholds are used
+        # ubiquitously
+        FPL_200 = (
+            self.df[
+                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.LOW_INCOME_THRESHOLD
+        )
+
+        did_we_meet_threshold = (
+            self.df[
+                field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        ) & FPL_200
+
+        self.df[field_names.FEMA_LOSS_RATE_LOW_INCOME] = did_we_meet_threshold
+
         climate_criteria = (
             (
                 self.df[
```

Comment:
I would strongly vote that we do *not* maintain redundant code with the same logic here. For instance, your logic only checks population loss, but this logic checks 3 FEMA fields. 

Can you just create `self.df[field_names.FEMA_LOSS_RATE_LOW_INCOME]` and the other two fields, and then the result returned is `self.df[field_names.FEMA_LOSS_RATE_LOW_INCOME] | <other field> | <other field>`?

Reply:
Yeah, exactly, that was my thought as well. I was planning to do that after your reviewed the core logic.

Reply:
@lucasmbrown-usds Just to be clear for FEMA there is only 1 of the 3 that is asked to be added as an indicator:

_Climate Change_:

_IF at or above the 90th percentile for FEMA’s expected population loss rate AND is low income_

Reply:
Yea, unfortunately this is a miscommunication, I think the ticket may be out of date. For now, make sure all 3 FEMA indicators are represented. We're still deliberating about whether to remove the other two.